### ISSUE TITLE
Roadmap of MMClassification

### ISSUE THREAD

**Issue by hellock** [open-mmlab/mmclassification#4]
We keep this issue open to collect feature requests from users and hear your voice. Our monthly release plan is also available here.

You can either:
1. Suggest a new feature by leaving a comment.
2. Vote for a feature request with 👍 or be against with 👎. (Remember that developers are busy and cannot respond to all feature requests, so vote for your most favorable one!)
3. Tell us that you would like to help implement one of the features in the list or review the PRs. (This is the greatest things to hear about!)

**Comment by GoldenAbel** [comment]
warm up and more pretrained models

**Comment by LeeJZh** [comment]
tricks like mix-up, auto-augmentation, and etc..

**Comment by Tshzzz** [comment]
convert model to onnx.

**Comment by SilvioGiancola** [comment]
EfficientNet

**Comment by Ironteen** [comment]
some Nerual Architecture Search algorithms which could be implement on one GPU, like ENAS

**Comment by soldier828** [comment]
multi-brach for multi-task learning

**Comment by LeeJZh** [comment]
> convert model to onnx.

I think pytorch 1.5 supports it. [torch.onnx](https://pytorch.org/docs/stable/onnx.html#)

**Comment by GoldenAbel** [comment]
darknet networks and data aug, like darknet53, mosaic, cutmix, mixup

**Comment by myopengit** [comment]
benchmark on cifar10 and cifar100

**Comment by zhongqiu1245** [comment]
Could you support HRNet?

**Comment by ousinkou** [comment]
Multi-label classification

**Comment by Kongsea** [comment]
More types of loss supported, such as `center loss`, `amsoftmax loss`, and so on.

**Comment by zhongqiu1245** [comment]
focal loss pls！

**Comment by hiyyg** [comment]
vovnet-v2, it is faster and more accurate than resnet in my experience.

**Comment by hiyyg** [comment]
HS-ResNet: Hierarchical-Split Block on Convolutional Neural Network
https://arxiv.org/abs/2010.07621
There is a reference repo: https://github.com/bobo0810/HS-ResNet, but no pretrained models are provided.
It would be nice if mmcls could include this and provide some pretrained models.

**Comment by zhangliukun** [comment]
multi-lable classification

**Comment by coderTazz** [comment]
The functionality of adding custom hooks like it is in mmdetection.

**Comment by cavalleria** [comment]
model_parallel

**Comment by NickChang97** [comment]
AUGMIX

**Comment by zhongqiu1245** [comment]
TTA
both of mmdet and mmseg support.

**Comment by limberc** [comment]
Why not just ensemble [timm](https://github.com/rwightman/pytorch-image-models)?

**Comment by mzr1996** [comment]
> 

We already have timm backbone wrapper.

**Comment by ke-dev** [comment]
Ensemble Learning
